helm/cluster-vsphere/templates/helpers/_workers.tpl

```diff
@@ -8,6 +8,16 @@ thumbprint: {{ $.Values.global.providerSpecific.vcenter.thumbprint }}
 {{- $pool = unset $pool "replicas" -}}
 {{- $pool = unset $pool "machineHealthCheck" -}}
 
+{{- $osName := include "cluster.os.name" $ }}
+{{- $osReleaseChannel := include "cluster.os.releaseChannel" $ }}
+{{- $osVersion := include "cluster.os.version" $ }}
+{{- $kubernetesVersion := include "cluster.component.kubernetes.version" $ }}
+{{- $osToolingVersion := include "cluster.os.tooling.version" $ }}
+
+{{- /* Modify $pool.template here */ -}}
+{{- $templateValue := printf "%s-%s-%s-kube-%s-tooling-%s-gs" $osName $osReleaseChannel $osVersion $kubernetesVersion $osToolingVersion -}}
+{{- $_ := set $pool "template" $templateValue -}}
+
```

Comment:
Maybe we should move the rendering of `datacenter`, `datastore`, `server` and `thumbprint` down here so it's in sync with the control plane rendering.